subprojects/gdk_rust/Cargo.toml

```diff
@@ -39,3 +39,7 @@ panic = 'abort'
 opt-level = 'z'
 codegen-units = 1
 incremental = false
+
+[patch.crates-io]
+bitcoin = { git = "https://github.com/shesek/rust-bitcoin", branch = "0.25.2-uint-serde", features = [ "use-serde" ] }
+electrum-client = { git = "https://github.com/shesek/rust-electrum-client", branch = "202011-custom-ssl-stream" }
```

Comment:
Do this need to be different from `electrum-client = "0.4.0-beta.1"`?

Reply:
It used to be needed but the necessary changes were merged upstream, I updated `gdk_rust/gdk_electrum/Cargo.toml` in https://github.com/Blockstream/gdk/pull/59/commits/a71aae7ec1ea645bd7361a9606a4ee082e0a7b0d but forgot about `gdk_rust/Cargo.toml`. Will update this too.

Reply:
Updated (forced-pushed over the previous commit)